saleor/graphql/file_upload/views.py

```diff
@@ -8,6 +9,18 @@
 
 class FileUploadGraphQLView(GraphQLView):
 
+    def dispatch(self, request, *args, **kwargs):
+        # Handle options method the GraphQlView restricts it.
+        if request.method.lower() == 'options':
```

Comment:
Is there really a case where the HTTP method is not passed as uppercase?

Reply:
It is more like my habit with keys from HTTP header. I will change that.